src/components/PasswordInput.tsx

```diff
@@ -29,8 +29,7 @@ const PasswordInput = <TFieldValues extends FieldValues>({
   };
 
   const validatePassword = (value: string) => {
-    const strongPasswordRegex =
-      /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+    const strongPasswordRegex = /((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*$/;
```

Comment:
create, please file in src  constants/regex.ts and move this constant this file and if you find other regex move them also

Reply:
I created a file for regex, you can look develop branch, it is inside config folder

Reply:
also we can use "zod" for validations instead of doing it manually